src/Hypergrid.js

```diff
@@ -1550,6 +1550,20 @@ Hypergrid.prototype = {
         });
         this.canvas.dispatchEvent(clickEvent);
     },
+    /**
+     * @memberOf Hypergrid.prototype
+     * @param {number} c - grid column index.
+     * @param {string[]} keys
+     */
+    fireSyntheticColumnSortEvent: function(c, keys) {
```

Comment:
this is fine, although the present task was just to moving sorting methods out, and in this case you have added one, so I'm just saying add it to the mix-in instead.